nototools/cmap_block_coverage.py

```diff
@@ -236,15 +252,20 @@ def _summarize_blocks(start, limit, defined_cps, cp_to_scripts, all_scripts):
   _summarize_block(block, block_count, defined_count, script_counts)
 
 
-def block_coverage(cmap_file, start=0, limit=0x20000, details=0, summary=False):
+def block_coverage(
+    cmap_file, start=0, limit=0x20000, only_scripts=None, details=0,
+    summary=False):
   data = cmap_data.read_cmap_data_file(cmap_file)
-  cp_to_scripts, all_scripts = _create_cp_to_scripts(data)
+  cp_to_scripts, all_scripts = _create_cp_to_scripts(data, only_scripts)
   defined_cps = unicode_data.defined_characters(version=9.0)
 
   if summary:
-    _summarize_blocks(start, limit, defined_cps, cp_to_scripts, all_scripts)
+    _summarize_blocks(
+        start, limit, defined_cps, cp_to_scripts, all_scripts, only_scripts)
```

Comment:
I think it was because I hadn't determined what if anything it should do.  I'm fixing the call site for now.